addons/docs/common/makePreset.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const common = require('./preset');
+
+module.exports = framework => {
+  const configModule = `${__dirname}/config-${framework}`;
+  const configContents = `
+const { addParameters } = require('@storybook/${framework}');
```

Comment:
Could we instead make it possible to import `addParameters` from `@storybook/core` or something to avoid this?